pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -340,6 +340,10 @@ def on_validation_end(self, trainer, pl_module):
             assert trainer.global_rank == 0, 'tried to make a checkpoint from non global_rank=0'
             self._save_model(filepath, trainer, pl_module)
 
+        if self.save_last:
```

Comment:
it seems that the code was just moved or am I missing something?

Reply:
yes, that's the fix! it makes a difference.